tests/haskell-load-tests.el

```diff
@@ -0,0 +1,85 @@
+;;; haskell-load-tests.el
+
+;;; Code:
+
+(require 'ert)
+(require 'haskell-test-utils)
+
+(require 'haskell-load)
+
+(defun insert-errors ()
+  (insert "import           Control.Applicativ\nimport           Data.Mayb\nimport           Data.String")
```

Comment:
One more nitpick: Can I have one space here between `import` and module names?

Counting spaces might be intimidating for future developers, too.